api/src/main/java/org/apache/iceberg/SortOrderBuilder.java

```diff
@@ -106,4 +106,12 @@ default R desc(Term term) {
    * @return this for method chaining
    */
   R desc(Term term, NullOrder nullOrder);
+
+  /**
+   * whether the Projection should consider case sensitivity on column names or not.
```

Comment:
Should be a bit more clear. 
"Whether to resolve column names in either a case-sensitive or insensitive way."?